package.json - entry main punta a file non esistente

## Versione di Bootstrap Italia

Bootstrap Italia v2.2.0.

## Comportamento atteso

Sto importando Bootstrap Italia v2.2.0 all'interno di una pipeline Webpack 5. Sto indicando al bundle di esportare l'oggetto `bootstrap` tramite il seguente snippet:

**webpack.config.js**
```
...
module.exports = {
    ...
    module: {
        ...
        rules: [
            ...
            {
                test: require.resolve("bootstrap-italia"),
                use: [{
                    loader: "expose-loader",
                    options: {
                        exposes: ["bootstrap"],
                    }
                }],
            }
        ]
    },
...
};
```
 
## Comportamento attuale

La build webpack fallisce alla riga `test: require.resolve("bootstrap-italia"),` riportando il seguente errore:

`EXEC : [webpack-cli] error : Cannot find module 'C:\myproject\Assets\node_modules\bootstrap-italia\dist\js\bootstrap-italia.min.js'. Please verify that the package.json has a valid "main" entry`

## Possibili soluzioni

In effetti il file menzionato nella entry main nel file package.json non esiste nel pacchetto:

https://github.com/italia/bootstrap-italia/blob/60792d17c53e1bc57a8bda87303e1c0e34d0c060/package.json#L24
![immagine](https://user-images.githubusercontent.com/16761934/211878156-6ab02120-080d-44a2-bd63-3c8a15c7131a.png)

La entry corretta sarebbe:

`"main": "dist/bootstrap-italia.esm.js",`

Attualmente il file contiene una entry chiamata `module` con il valore corretto, ma non mi risulta che sia standard ES (cfr. https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for). Veniva usata da Webpack 2, probabilmente Webpack 5 non la considera più.

Se serve posso fornire PR.

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
   "description": "Bootstrap Italia è un tema Bootstrap 5 per la creazione di applicazioni web nel pieno rispetto delle linee guida di design per i siti internet e i servizi digitali della PA",
   "homepage": "https://italia.github.io/bootstrap-italia/",
   "style": "dist/css/bootstrap-italia.min.css",
-  "main": "dist/js/bootstrap-italia.min.js",
+  "main": "dist/js/bootstrap-italia.bundle.min.js",
   "scripts": {
     "start": "concurrently \"npm run dev-code\" \"npm run dev-docs\" \"npm run start-bs\"",
     "build": "rollup -c",

